lib/tds.js

```diff
@@ -307,28 +308,25 @@ async function generateDNRRulesForTrackerEntry (
 
         priority += TRACKER_RULE_PRIORITY_INCREMENT
 
-        const newRuleTemplate = {
-            priority,
-            actionType: ruleAction,
-            redirect: redirectAction,
-            urlFilter,
-            regexFilter,
-            matchCase,
-            requestDomains,
-            excludedInitiatorDomains,
-            resourceTypes
-        }
-        const newRule = generateDNRRule(newRuleTemplate)
-
-        if (customRuleAction) {
-            // if a custom action, map the action to the specified default
-            // and add its inverse to the list of custom action rules
-            const inverseRule = { ...newRuleTemplate }
-            inverseRule.actionType = customRuleAction.inverseAction
-            newRule[inverseCustomActionSymbol] = { customAction: trackerEntryRules[i].action, ...generateDNRRule(inverseRule) }
-        }
+        {
```

Comment:
Because we define an intermediate variable `newRule` that's not used later. The block makes it clear that the variable only is used (and even exists) in its scope.

Reply:
Gotcha if its just for block scoping that's fine, tho not a pattern we've been using in the ext.